lib/order.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'menu'
+require 'rubygems'
+require 'twilio-ruby'
+require 'time'
+# require 'sinatra'
+
+# post '/message' do
+#   number = params['From']
+#   body = params['Body']
+
+#   content_type 'text/xml'
+#   "<Response>
+#     <Message>
+#       Hello #{number}. You said: #{body}
+#     </Message>
+#   </Response>"
+# end
+
+class Order
+
+  attr_reader :order, :order_total, :simp_order
+
+  def initialize
+
+    @menu = Menu.new
+    @order = []
+    @order_total = 0
+    @simp_order = ""
+
+    account_sid = ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+
+  end
+
+  def pick_item(item, quantity)
+
+    fail "Please choose from the menu" unless @menu.menu.has_key?(item)
+    quantity.times { @order << item }
+
+  end
+  
+  def order_value
+
+    count = 0
+    @order.each do |order_item|
+      if @menu.menu.has_key?(order_item)
+        count += @menu.menu[order_item]
+      end
+    end
+    @order_total + count
+
+  end
+
+  def arrival_time
+
+    (Time.now + 1 * 60 * 60).strftime("%k:%M")
+
+  end
+
+  def place_order
+
+    send_order_text
+    "Thanks! Your order has been received and will be with you by #{arrival_time}"
+
+  end
+
+  def order_summary
+
+    "Your order so far is: #{simplify_order}"
+
+  end
+
```

Comment:
I think this should be re-written so as not to be hard coded, but I like that you added a function that states the quantity of each item - that's not something I did.